iica-catie.csl

```diff
@@ -208,6 +199,9 @@
       <name form="short" and="text" delimiter=", " delimiter-precedes-last="never" initialize-with=". "/>
     </names>
   </macro>
+  <macro name="Editor">
+    <text term="editor" form="short" suffix="(ed.)."/>
+  </macro>
```

Comment:
This is incorrect. You always want to print the names of the "editor" variable through `<names/>`, not `<text/>`. See the "author-citation" macro directly above as an example (or look at the "editor" macro of the original style).

Reply:
ok